UnitTests/Drivers/ConsoleDriverTests.cs

```diff
@@ -440,8 +440,6 @@ public void MakePrintable_Does_Not_Convert_Ansi_Chars_To_Unicode (uint code)
 			Assert.Equal (code, actual.Value);
 		}
 
-		private static object packetLock = new object ();
```

Comment:
Without this lock sometimes the `TestVKPacket` fails. Do you think it isn't actually necessary?

Reply:
This code was accidentally deleted during a merge. I'll add it back.